### PR TITLE
Update gallery.yml with introPy book

### PR DESCRIPTION
**Pull request by lukassnoek**
Added the Jupyter Book of our 2-week introductory Python and PsychoPy course taught at the University of Amsterdam.

### CODE REVIEW

File: docs/gallery.yml [hunk 1]
```diff
@@ -85,3 +85,7 @@
   website: https://altair-viz.github.io/altair-tutorial/README.html
   repository: https://github.com/altair-viz/altair-tutorial
   image: https://altair-viz.github.io/altair-tutorial/_static/altair-logo.png
+- name: introPy: an introduction to Python and PsychoPy
```
Comment (choldgraf) [inline docs/gallery.yml:88]:
```suggestion
- name: "introPy: an introduction to Python and PsychoPy"
```

Comment (lukassnoek) [reply docs/gallery.yml]: Oops, sorry!

**Comment by choldgraf** [comment]
hey there - the image in your thumbnail doesn't seem to be working, maybe it is not the correct github path (which needs to be a link to a *raw* file)? https://executablebookproject--217.org.readthedocs.build/en/217/gallery.html

**Comment by lukassnoek** [comment]
Ah, you're completely right. It should be: `https://raw.githubusercontent.com/lukassnoek/introPy/master/intropy/img/logo.png`.
Should I make a new PR?

**Comment by lukassnoek** [comment]
I'll make a new PR.

**Pull request closed without merging**